host/4/bookworm/python/python312/python312-appservice.Dockerfile

```diff
@@ -0,0 +1,112 @@
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V3 && \
+    unzip /$EXTENSION_BUNDLE_FILENAME_V3 -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V3 && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME_V3 &&\
+    EXTENSION_BUNDLE_VERSION_V4=4.12.0 && \
+    EXTENSION_BUNDLE_FILENAME_V4=Microsoft.Azure.Functions.ExtensionBundle.${EXTENSION_BUNDLE_VERSION_V4}_linux-x64.zip && \
+    wget https://functionscdn.azureedge.net/public/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V4/$EXTENSION_BUNDLE_FILENAME_V4 && \
+    mkdir -p /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V4 && \
+    unzip /$EXTENSION_BUNDLE_FILENAME_V4 -d /FuncExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle/$EXTENSION_BUNDLE_VERSION_V4 && \
+    rm -f /$EXTENSION_BUNDLE_FILENAME_V4 &&\
+    find /FuncExtensionBundles/ -type f -exec chmod 644 {} \;
+
+FROM mcr.microsoft.com/oryx/python:3.12-debian-bookworm AS python
+
+# Install Python dependencies
+# MS SQL related packages: unixodbc msodbcsql18 mssql-tools
+# .NET Core dependencies: --no-install-recommends ca-certificates libc6 libgcc1 libgssapi-krb5-2 libicu67 libssl1.1 libstdc++6 zlib1g
+# OpenCV dependencies:libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb
+# binutils: binutils
+# OpenMP dependencies: libgomp1 && \
+# Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157 : libc-dev
+# Azure ML dependencies: liblttng-ust0
+RUN curl -fsSL https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o /usr/share/keyrings/microsoft-prod.gpg && \
+    apt-get update && \
+    apt-get install -y wget apt-transport-https curl gnupg2 locales rpm && \
+    echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections && \
+    curl https://packages.microsoft.com/keys/microsoft.asc | tee /etc/apt/trusted.gpg.d/microsoft.asc && \ 
+    echo "deb [arch=amd64] https://packages.microsoft.com/debian/12/prod bookworm main" | tee /etc/apt/sources.list.d/mssql-release.list && \
+    # Needed for libss1.0.0 and in turn MS SQL
+    echo 'deb http://security.debian.org/debian-security bookworm-security main' >> /etc/apt/sources.list && \
+    curl https://packages.microsoft.com/config/debian/12/prod.list | tee /etc/apt/sources.list.d/mssql-release.list && \
+    # install MS SQL related packages.pinned version in PR # 1012.
+    echo 'en_US.UTF-8 UTF-8' > /etc/locale.gen && \
+    locale-gen && \
+    apt-get update && \
+    ACCEPT_EULA=Y apt-get install -y unixodbc msodbcsql18 mssql-tools18 &&\
+    apt-get install -y --no-install-recommends ca-certificates \
+    libc6 libgcc1 libgssapi-krb5-2 libicu72 libssl3 libstdc++6 zlib1g &&\
+    apt-get install -y libglib2.0-0 libsm6 libxext6 libxrender-dev xvfb binutils \
+    binutils libgomp1 libc-dev liblttng-ust1 && \
```

Comment:
Do you know what this binutils statement does? If it is an install we should remove it but its just got a \ in front of it so I don't really undersatnd what this line means

Reply:
binutils installs a collection of binaries. Not sure if its required by some dependency or we just install it since some customers need it. But either way, since its in other images, lets keep it so that there is not change if some users need it. I removed duplicate install statements in the image.